Makefile

```diff
@@ -1,9 +1,14 @@
 # Ensure go bin path is in path (especially for CI)
 PATH := $(PATH):$(GOPATH)/bin
 
+# include script to possibly set a crypto flag for older machines
+include crypto_adx_flag.mk
+
+CGO_FLAG := CGO_CFLAGS=$(CRYPTO_FLAG)
+
 .PHONY: test
 test:
-	GO111MODULE=on go test -coverprofile=coverage.txt ./...
+	GO111MODULE=on $(CGO_FLAG) go test -coverprofile=coverage.txt ./...
```

Comment:
This flag basically disables fast code using ADX instructions. 
If the machine is old, that is necessary. If the machine is new, either way works, we just get slower times with a set flag. 

If you think the `.mk` file is disruptive, we can also remove it and force the build to always run without the fast code. This would work for all machines but always give slower times (for BLS operations only, so not a big impact)